modules/monitoring-deploying-a-sample-service-for-cluster-observability-operator.adoc

```diff
@@ -0,0 +1,91 @@
+// Module included in the following assemblies:
+//
+// monitoring/cluster-observability-operator/configuring-the-cluster-observability-operator-to-monitor-a-service.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="deploying-a-sample-service-for-cluster-observability-operator_{context}"]
+= Deploying a sample service for Cluster Observability Operator
+
+This sample configuration deploys a sample service named `prometheus-coo-example-app` in the user-defined `ns1-coo` project. 
+The service exposes the custom `version` metric.
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` cluster role or the `monitoring-edit` cluster role.
```

Comment:
A user with cluster role `monitoring-edit` has no enough permission to create APP service and can only manage the following resources
```
 - servicemonitors
  - podmonitors
  - prometheusrules

Reply:
admin role of specific namespace can create app service in it

Reply:
+1 I think we might want to avoid referencing roles that are created by CMO. That just creates unnecessary coupling.
A namespace admin role would look like this
```
apiVersion: rbac.authorization.k8s.io/v1
kind: Role
metadata:
 name: read-write
 namespace: ns1-coo
rules:
- apiGroups:
 - ""
 resources: ["*"]
 verbs:
 - get
 - list
 - watch
 - create
 - update
 - patch
 - delete
```
Though maybe there is a convention is the docs already?

Reply:
@jan--f @lihongyan1 -

What do you think of this wording:

`* You have access to the cluster as a user with the `cluster-admin` cluster role or as a user with administrative permissions for the namespace.`

Reply:
The changes look good.